From 7241f8b110cf3e801c20343d8c9e37f750ac0203 Mon Sep 17 00:00:00 2001
From: Ray Strode <rstrode@redhat.com>
Date: Thu, 26 Feb 2015 10:07:28 -0500
Subject: [PATCH 162/315] session: add more debug spew about session type

This should give a clearer picture about what session type
is getting picked when.

https://bugzilla.gnome.org/show_bug.cgi?id=745235
---
 daemon/gdm-session.c | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/daemon/gdm-session.c b/daemon/gdm-session.c
index 3303a9f5..ad42b5b8 100644
--- a/daemon/gdm-session.c
+++ b/daemon/gdm-session.c
@@ -2968,6 +2968,8 @@ gdm_session_select_session_type (GdmSession *self,
         GHashTableIter iter;
         gpointer key, value;
 
+        g_debug ("GdmSession: selecting session type '%s'", text);
+
         g_hash_table_iter_init (&iter, self->priv->conversations);
         while (g_hash_table_iter_next (&iter, &key, &value)) {
                 GdmSessionConversation *conversation;
@@ -2988,6 +2990,8 @@ gdm_session_select_session (GdmSession *self,
         gpointer key, value;
         gboolean is_wayland_session = FALSE;
 
+        g_debug ("GdmSession: selecting session '%s'", text);
+
         g_free (self->priv->selected_session);
         self->priv->selected_session = g_strdup (text);
 
@@ -3101,6 +3105,8 @@ static void
 set_session_type (GdmSession *self,
                   const char *session_type)
 {
+
+        g_debug ("GdmSession: setting session to type '%s'", session_type? session_type : "");
         g_free (self->priv->session_type);
         self->priv->session_type = g_strdup (session_type);
 }
-- 
2.15.1

